### PR TITLE
Clean up tests for -3 run

### PR DESCRIPTION
**Pull request by nanjekyejoannah**
Test clean up before fix to `ceval`.

### CODE REVIEW

File: Lib/test/string_tests.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 import unittest, string, sys, struct
 from test import test_support
 from UserList import UserList
+import sys
```
Comment (ltratt) [inline Lib/test/string_tests.py]: I think `sys` is already imported on line 5?

Comment (nanjekyejoannah) [reply Lib/test/string_tests.py]: I have addressed this.

**Comment by nanjekyejoannah** [comment]
This PR includes some reverts I did to fix some tests. I decided to just open the new PRs after.

**Comment by ltratt** [comment]
Please squash.

**Comment by nanjekyejoannah** [comment]
Squashed

**Comment by nanjekyejoannah** [comment]
bors try

**Pull request merged**